src/main/kotlin/ProblemA.kt

```diff
@@ -1,9 +1,24 @@
 class ProblemA {
-
     fun main() {
-
         //ваша реализация в методе main
-
+        var t = readInt()
+        while (t > 0) {
+            val stringL = readLongArray(3)
+            val a = stringL[0]
+            val b = stringL[1]
+            var k = stringL[2]
+            var x : Long = 0
+            while (k > 0) {
+                if (k%2 != 0){
```

Comment:
результат выражения k%2 является типом Long, который затем сравнивается с 0 (это тип Int)
чтобы исправить, нужно либо привести k%2 к Int, либо использовать 0 типа Long (определяется записью 0L)
```suggestion
                if (k%2 != 0 L){
```